### ISSUE TITLE
Create build process video

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#63]
## User story
1. As a SD
2. I want to create a build process video
3. So that I can demonstrate the build process.

## Acceptance criteria
*  The video demonstrates that you mastered the build proces
*  The video shows the terminal window
*  Atleast consists of the four main steps i.e Clone, Build, Test & Run

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
